grakn-engine/src/main/java/ai/grakn/engine/user/SystemKeyspaceUsers.java

```diff
@@ -55,22 +55,22 @@
      */
     @Override
     public boolean addUser(Json userJson) {
-        final Var user = var().isa(USER_ENTITY);
+        final Var[] user = {var().isa(USER_ENTITY)};
```

Comment:
Uhm why change to an array if you only ever going to use the first element of the array?